_includes/gsoc-howto.html

```diff
@@ -12,11 +12,14 @@ <h2 class="section-heading">How does it work?</h2>
           </p>
           <p>
             <i class="fa fa-search fa-fw icon-mentoring"></i>
-            Select one of our mentoring <a href="#projects">projects</a>.
+            Select one of our mentoring <a href="#projects">projects</a> and follow the get started instructions in the project description.
           </p>
           <p>
             <i class="fa fa-comments fa-fw icon-mentoring"></i>
-            Get in touch with the mentor, follow the get started instructions in the project description.
+            Ask questions in our <a href="mailto: opensuse-project@opensuse.org">mailing list</a>,
+            <a href="" target="_blank">Twitter</a> and
```

Comment:
Isn't it missing the twitter link?

Reply:
ouch! 🙈

Reply:
I would also not write twitter but the twitter account directly (@...)